tests/test_geomopt_cli.py

```diff
@@ -124,6 +131,10 @@ def test_fully_opt_and_vectors(tmp_path, caplog):
         assert result.exit_code == 0
         assert "Using filter" in caplog.text
 
+        atoms = read(results_path)
```

Comment:
do we want read atoms in here?

Reply:
As in, rather than the utils read/delete file function?

The results are in a `tmp_path`, so just reading them is fine, and it's cleaned up

Reply:
cool